packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -254,7 +254,7 @@ function isProd(config) {
 				preload: 'media',
 				pruneSource: false,
 				logLevel: 'silent',
-				additionalStylesheets: ['*.css'],
+				additionalStylesheets: ['route-*.css'],
```

Comment:
Critters has always duplicated the styles in `bundle.css` due to it too matching this pattern, but historically has deduped through using `cssnano` as means to compress. `cssnano` is no longer in use, so we're getting duplicate CSS. See [critters/88](https://github.com/GoogleChromeLabs/critters/issues/88).

Not yet sure if this is a regression to be fixed in Critters or not, but we can get around it by only matching chunk split CSS, which I believe is the intention anyhow?